lib/ansible/modules/network/ios/_ios_template.py

```diff
@@ -33,8 +34,10 @@
     by evaluating the current running-config and only pushing configuration
     commands that are not already configured.  The config source can
     be a set of commands or a template.
+notes:
+  - Provider arguments are no longer supported.  Network tasks should now
```

Comment:
Might be worth updating https://github.com/ansible/ansible/blob/devel/CHANGELOG.md in this PR as well to made that clear